deps.versions.toml

```diff
@@ -1,25 +1,25 @@
 [versions]
 
 decompose = "2.0.1-compose-experimental"
-kotlin = "1.8.20"
+kotlin = "1.8.21"
 essenty = "1.1.0"
 parcelizeDarwin = "0.1.4"
 reaktive = "1.2.3"
 junit = "4.13.2"
-jetbrainsCompose = "1.4.0"
+jetbrainsCompose = "1.5.0-beta01"
 jetbrainsKotlinWrappers = "1.0.0-pre.478"
-jetbrainsKotlinxCoroutines = "1.6.4"
+jetbrainsKotlinxCoroutines = "1.7.1"
```

Comment:
Same here. If this is unrelated to the PR and just on-the-fly, it's better to do it separately, describing the benefits. Same for all versions below.

Reply:
This is very likely not necessary and I will revert the changes. :)